app/controllers/events/volunteerRegisterEvent.py

```diff
@@ -0,0 +1,34 @@
+from app.models import*
+from app.models.event import Event
+from app.models.eventParticipant import EventParticipant
+from app.models.user import User
+from app.controllers.events.programEligibility import isEligibleForProgram
+import pytest
+
+def volunteerRegister(userid,  eventid):
+    """
+    Lets the user register for an event if they are eligible
+    :param userid: indicates the id that the user has
+    :param eventid: indicates the id that the event has
+    :return: a message that indicates what the user has registered for if eligible;
+    otherwise lets the user know that they are not eligible for registration
+    """
+
+    user = User.get(User.username == userid)
+    event = Event.get(Event.id == eventid)
+    program = Event.select(Event.program).where(Event.id == eventid)
```

Comment:
You are still using the argument eventid, not `event` that you just got in the line before

Reply:
Replaced the eventid argument with the event argument